HighwayPatrol.cpp

```diff
@@ -27,7 +33,17 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     if( willArrest )
     {
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+
+        std::cout << name << ": YOU IN THE [ "; 
+        
+        if( dynamic_cast<Car*>(v) )
+            std::cout << "Car";
+        if( dynamic_cast<Motorcycle*>(v) )
+                std::cout << "Motorcycle";
+        if( dynamic_cast<SemiTruck*>(v) )
+            std::cout << "Truck";
+        
+        std::cout << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
Interesting solution.
Most people use `std::string` locally to fill in between the `[ ]`
- [ ] can you revise and use std::string so the `std::cout` statement isn't broken up?